lib/restbase.js

```diff
@@ -56,7 +60,7 @@ RESTBase.prototype.request = function (req) {
             ((req.uri.params && req.uri.params.api === 'sys')
              // TODO: Remove once params.api is reliable
              || (req.uri.path.length > 1 && req.uri.path[1] === 'sys'))) {
-        return Promise.reject(new rbUtil.HTTPError({
+        return Promise.reject(new HTTPError({
```

Comment:
It could, but I think we should really be very specific in our errors. For the generic ones we could however set up a default `type` and `title` [on the way out](https://github.com/wikimedia/restbase/blob/817cd5037d8d5b126deda3c1cdcdea0134c84d1f/lib/server.js#L39).